<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPanel</class>
 <widget class="QWidget" name="UserPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserPanel</string>
  </property>
  <widget class="QTableWidget" name="userOrderTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>1021</width>
     <height>281</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backwardButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: none; border-radius: 3px; color: black; } 
QPushButton:hover { background-color:rgb(233, 182, 163); }
</string>
   </property>
   <property name="text">
    <string>&lt;-返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/touxiang.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="phoneLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ID_Label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份证号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backwardButton_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: none; border-radius: 3px; color: black; } 
QPushButton:hover { background-color:rgb(233, 182, 163); }
</string>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePersonInfoButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改个人信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: none; border-radius: 3px; color: black; } 
QPushButton:hover { background-color:rgb(233, 182, 163); }
</string>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新订单信息</string>
   </property>
  </widget>
  <zorder>userOrderTable</zorder>
  <zorder>nameLabel</zorder>
  <zorder>accountLabel</zorder>
  <zorder>backwardButton</zorder>
  <zorder>phoneLabel</zorder>
  <zorder>ID_Label</zorder>
  <zorder>backwardButton_2</zorder>
  <zorder>changePersonInfoButton</zorder>
  <zorder>logoutButton</zorder>
  <zorder>refreshButton</zorder>
  <zorder>iconLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
